net-misc/astmanproxy/astmanproxy-1.28.2.ebuild

```diff
@@ -1,25 +1,25 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit base flag-o-matic multilib toolchain-funcs
+inherit flag-o-matic multilib toolchain-funcs
```

Comment:
`multilib` can go (`get_libdir` is part of EAPI 6+)

Reply:
Ah, that's right, dropped